src/Compilers/CSharp/Test/Semantic/Semantics/OperatorTests.cs

```diff
@@ -8124,10 +8120,10 @@ struct TestStr
                         Assert.Equal("System.Boolean System.Object.op_Inequality(System.Object left, System.Object right)", info1.Symbol.ToTestDisplayString());
                         break;
                     case 8:
-                        Assert.Equal("System.Boolean System.Int32.op_Equality(System.Int32 left, System.Int32 right)", info1.Symbol.ToTestDisplayString());
+                        Assert.Equal("System.Boolean System.Int32?.op_Equality(System.Int32? left, System.Int32? right)", info1.Symbol.ToTestDisplayString());
```

Comment:
>System.Int32? [](start = 53, length = 13)

Containing type change feels unexpected.

Reply:
From IDE perspective, i would be ok with this.  @sharwell for thoughts as well.  

Note: it has been weird IMO that we show the underlying non-lifted operators.  Iknow i've been confused by this myself when using quick-info when working with nullable values and operators.  So i do appreciate improvements here.